website/content/v1.3/introduction/getting-started.md

```diff
@@ -167,10 +167,18 @@ kernel commandline parameters.
 See [required kernel parameters]({{< relref "../reference/kernel" >}}).
 If creating [EC2 kubernetes clusters]({{< relref "../talos-guides/install/cloud-platforms/aws/" >}}), the configuration file can be passed in as `--user-data` to the `aws ec2 run-instances` command.
 
-In any case, we need to generate the configuration which is to be provided:
+In any case, we need to generate the configuration which is to be provided.
+We start with generating a secrets bundle which should be saved in a secure location and used
+to generate machine or client configuration at any time:
 
 ```sh
-  talosctl gen config cluster-name cluster-endpoint
+talosctl gen secrets -o secrets.yaml
+```
+
+Now, we can generate the machine configuration for each node:
+
+```sh
+talosctl gen config --with-secrets secrets.yaml cluster-name cluster-endpoint
```

Comment:
```suggestion
talosctl gen config --with-secrets secrets.yaml <cluster-name> <cluster-endpoint>
```

Nit, but I prefer wrapping these in brackets to make it clear that they aren't just to be copied/pasted and need to be edited first.